articles/storage/common/storage-network-security.md

```diff
@@ -90,7 +90,11 @@ After you apply network rules, they're enforced for all requests. SAS tokens tha
 
 [Network Security Perimeter](../../private-link/network-security-perimeter-concepts.md) (preview) allows organizations to define a logical network isolation boundary for PaaS resources (for example, Azure Blob Storage and SQL Database) that are deployed outside their virtual networks. The feature restricts public network access to PaaS resources outside the perimeter. However, you can exempt access by using explicit access rules for public inbound and outbound traffic. By design, access to a storage account from within a Network Security Perimeter takes the highest precedence over other network access restrictions.
 
-Currently, Network Security Perimeter is in public preview for Azure Blobs, Azure Files (REST), Azure Tables, and Azure Queues. See [Transition to a Network Security Perimeter](../../private-link/network-security-perimeter-transition.md).
+Currently, Network Security Perimeter is in public preview for Azure Blobs, Azure Files (REST), Azure Tables, and Azure Queues. See [Transition to a Network Security Perimeter](../../private-link/network-security-perimeter-transition.md). 
+
+The list of services that have been onboarded to NSP can be found [here](../../private-link/network-security-perimeter-concepts#onboarded-private-link-resources). 
```

Comment:
```suggestion
The list of services that have been onboarded to network security perimeter can be found [here](../../private-link/network-security-perimeter-concepts#onboarded-private-link-resources). 
```

Reply:
NSP is not an acronym used for this capability. Typically, to avoid the risk of trademark infringement, we use only approved acronyms for products and services (rarely a feature). I would defer to the private link folks as they are technically the owner of this capability. They are not using "NSP" in those docs - https://learn.microsoft.com/en-us/azure/private-link/network-security-perimeter-concepts.